src/Stratis.Bitcoin.Features.Api/ApiSettings.cs

```diff
@@ -40,9 +41,10 @@ public ApiSettings(Action<ApiSettings> callback)
         /// <summary>
         /// Loads the API related settings from the application configuration.
         /// </summary>
```

Comment:
The `nodeSettings` parameters has been deleted, is that intentional?

Reply:
yes. It is introduced via `ApiFeature.LoadConfiguration`.